backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/bpartner/bpartnercomposite/JsonRetrieverService.java

```diff
@@ -799,4 +824,32 @@ private static JsonResponseContactPosition toJson(@Nullable final Job job)
 				.active(job.isActive())
 				.build();
 	}
+
+	@NonNull
+	private JsonProductBPartner toJsonProductBPartner(final I_C_BPartner_Product record)
+	{
+		return JsonProductBPartner.builder()
+				.bpartnerId(JsonMetasfreshId.of(record.getC_BPartner_ID()))
+				.productId(JsonMetasfreshId.of(record.getM_Product_ID()))
+				//
+				.productNo(record.getProductNo())
+				.productName(record.getProductName())
+				.productDescription(record.getProductDescription())
+				.productCategory(record.getProductCategory())
+				//
+				.ean(record.getUPC())
```

Comment:
```suggestion
				.ean(record.getEAN_CU())
```


..or do i miss sth?

Reply:
There's a similar method in GetProductsCommand: de.metas.rest_api.v2.product.command.GetProductsCommand#toJsonProductBPartner
I've inspired from there, that's why I used UPC, but I can change it to EAN_CU if needed

Reply:
Also, from the current description seems to make sense to use it as `ean`

![image](https://user-images.githubusercontent.com/83770997/152224450-cf5aa112-7e0c-4950-a0f7-b787abddbb19.png)

Reply:
i see the point, but that evidence is misleading..thx for the hint regarding GetProductsCommand..i'm fixing it there as well

Reply:
ah, no..there in M_Product we don't have EAN..ther it's kindof right (the column naming is unfortunate though..)